scripts/aggregate-themes/README.md

```diff
@@ -61,4 +60,58 @@ The `theme` option accepts the string name of a default theme. The default theme
 
 #### Scoped (Optional)
 
-The `scoped` option accepts an array of theme names to aggregate. Only the `scoped-theme.scss` files will be aggregated.
\ No newline at end of file
+The `scoped` option accepts an array of theme names to aggregate. Only the `scoped-theme.scss` files will be aggregated.
+
+## Getting Started - Generate Root and Scoped Themes
```

Comment:
I think we should flip these Getting Started sections so this one appears first in the ReadMe so it's the first thing new interested parties see and is presented as the primary recommendation.

Reply:
The intro may need a little tweaking too, to remove the "alternatively" and establish the expected setup

Reply:
updated - take another look https://github.com/cerner/terra-toolkit/pull/324/commits/db42599dd9489c0198a0eb99e637cfe9d88b8ec8#diff-a385b55d94aefec6207446ff49b50d96R5